src/test/java/codeu/controller/ConversationServletTest.java

```diff
@@ -73,8 +73,8 @@ public void testDoGet() throws IOException, ServletException {
 
     conversationServlet.doGet(mockRequest, mockResponse);
 
-    Mockito.verify(mockRequest).setAttribute("conversations", fakeConversationList);
-    Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+//    Mockito.verify(mockRequest).setAttribute("conversations", fakeConversationList);
```

Comment:
Is there a reason this has been commented out?

Reply:
I was accidentally testing the ConversationServlet with the new ConversationStore.getConversationsForUser(String username) method so I temporarily commented these out and forgot to uncomment them, thanks for pointing this out! Fixed in the new commit